apps/src/templates/progress/ProgressBubble.jsx

```diff
@@ -104,4 +106,9 @@ const ProgressBubble = React.createClass({
 // connector between bubbles
 ProgressBubble.height = DOT_SIZE + styles.main.marginTop + styles.main.marginBottom;
 
-export default Radium(ProgressBubble);
+// If progressBubbles is enabled, use our NewProgressBubble instead
+let ExportedProgressBubble = Radium(ProgressBubble);
+if (experiments.isEnabled('progressBubbles')) {
+  ExportedProgressBubble = NewProgressBubble;
+}
+export default ExportedProgressBubble;
```

Comment:
I wanted to be able to be able to work on our new ProgressBubble class with minimal risk of regression existing bubbles. This is how I'm trying to accomplish that.